### PR TITLE
Make lint/type tests available without docker build

### PR DESCRIPTION
**Pull request by kchygoe**
## What this patch does to fix the issue.
This change makes lint/type tests executed on container.
Docker build is no longer required for these tests.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -12,13 +12,18 @@ deps:
...
 
 .PHONY: install-gpu
 install-gpu: install
-	pip install -e .[gpu]
-	pip install -e .[dist]
+	pip3 install -e .[gpu]
+	pip3 install -e .[dist]
 
 .PHONY: lint
 lint:
```
Comment (tfujiwar) [inline Makefile:29]: Maybe we can remove the target `lint` now.

Comment (kchygoe) [reply Makefile:29]: it would be better to remain target `lint` and remove `test-blueoil-pep8`

Comment (kchygoe) [reply Makefile]: Fixed target names from `test-blueoil-pep8` to `test-lint` and from `test-blueoil-mypy` to `test-mypy`

**Pull request merged**